### PR TITLE
fix: update DevCard Contribution term to Repo

### LINKED ISSUE #1318: feat: Contributions should be Repos
The label is contributions, but it should be Contributed repos.

Screen Shot 2023-06-28 at 9 35 43 PM

https://github.com/open-sauced/insights/blob/beta/pages/api/card.png.tsx#L63

### PR DESCRIPTION
**Pull request by babblebey**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR updates the term "Contribution" to "Repo" on the DevCard feature.

## Related Tickets & Documents

fixes #1318 

## Mobile & Desktop Screenshots/Recordings

**BEFORE**

Greater than 1 Repo
screenshot-localhost_3000-2023 06 30-15_33_31

1 or Less Repo
screenshot-localhost_3000-2023 06 30-15_35_17

**AFTER**

Greater than 1 Repo
screenshot-localhost_3000-2023 06 30-16_19_21

1 or Less Repo
screenshot-localhost_3000-2023 06 30-16_09_45

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
NA

### CODE REVIEW

**Comment by brandonroberts** [comment]
Thanks @babblebey. Will you provide some before/after screenshots?

File: pages/api/card.png.tsx [hunk 1]
```diff
@@ -139,10 +139,10 @@ export default async function handler(request: NextRequest) {
                   </div>
                   <div tw="flex flex-col text-center items-center">
                     <div tw="flex font-black" style={{ fontSize: size(60) }}>
-                      {contributions}
+                      {repos}
                     </div>
                     <div tw="flex" style={{ fontSize: size(12) }}>
-                      Contributions
+                      {repos > 1 ? "Repos" : "Repo"}
```
Comment (NsdHSO) [inline pages/api/card.png.tsx:145]:
```suggestion
                      {repos > 1 ? "Contributions" : "Contributed"}
```

**Comment by babblebey** [comment]
> Thanks @babblebey. Will you provide some before/after screenshots? It helps for a quicker review

Done @brandonroberts

Kindly note, I committed some more changes to the branch.

File: pages/api/card.png.tsx (hunk 1 above)
Comment (brandonroberts) [inline pages/api/card.png.tsx:145]: Should show "Repos" if 0 or greater than 1

```suggestion
                      {repos === 1 ? "Repo" : "Repos"}
```

Comment (babblebey) [reply pages/api/card.png.tsx:145]: I'm on it

File: components/molecules/DevCard/dev-card.tsx [hunk 2]
```diff
@@ -184,8 +184,8 @@ export default function DevCard(props: DevCardProps) {
                   <div className="text-xs">PRs created</div>
                 </div>
                 <div className="text-center">
-                  <div className="text-6xl font-black">{props.isLoading ? "-" : props.contributions}</div>
-                  <div className="text-xs">Contributions</div>
+                  <div className="text-6xl font-black">{props.isLoading ? "-" : props.repos}</div>
+                  <div className="text-xs">{props.repos! <= 1 ? "Repo" : "Repos"}</div>
```
Comment (OgDev-01) [inline components/molecules/DevCard/dev-card.tsx:188]: This needs to be exact
```suggestion
                  <div className="text-xs">{props.repos! === 1 ? "Repo" : "Repos"}</div>
```

File: pages/api/card.png.tsx [hunk 3]
```diff
@@ -139,10 +139,10 @@ export default async function handler(request: NextRequest) {
                   </div>
                   <div tw="flex flex-col text-center items-center">
                     <div tw="flex font-black" style={{ fontSize: size(60) }}>
-                      {contributions}
+                      {repos}
                     </div>
                     <div tw="flex" style={{ fontSize: size(12) }}>
-                      Contributions
+                      {repos <= 1 ? "Repo" : "Repos"}
```
Comment (OgDev-01) [inline pages/api/card.png.tsx:145]: Same here
```suggestion
                      {repos === 1 ? "Repo" : "Repos"}
```

**Review by OgDev-01 — CHANGES_REQUESTED** [review]

**Comment by OgDev-01** [comment]
Hey @babblebey, will you check the build error?

>hint `npm run build`

File: components/molecules/DevCard/dev-card.tsx (hunk 2 above)
Comment (babblebey) [reply components/molecules/DevCard/dev-card.tsx:188]: Very well.

But, this does mean that in cases where the `repos` value is `0`, the term takes its plural form i.e. "0 Repos"

Is that fine?

**Comment by babblebey** [comment]
> Hey @babblebey, will you check the build error?
> 
> >hint `npm run build`

Will do @OgDev-01

**Comment by babblebey** [comment]
> Hey @babblebey, will you check the build error?
> 
> > hint `npm run build`

@OgDev-01 

I did, found an error and fixed the term `repos` in place of `contributor` in the `DevCard` story.

Thank you || O seun || Danke

**Review by brandonroberts — APPROVED** [review]

**Review by OgDev-01 — APPROVED** [review]
looks good 🍕 🍕

**Pull request merged**